File: prime-number-checker/prime.js
// Importing the necessary functions from modules.
const ps = require("prompt-sync")()
prompt = ps

// Getting the necessary values from the user.
let num = parseInt(prompt("Enter the number you want to check to be a prime number or not: "))
let nonDivisibles = 0

if (num > 1) {

    for (divisor = 2; divisor < num; divisor ++) {

        if (num % divisor == 0) {

            console.log("The number is composite.")
            break

        } else nonDivisibles ++

    }

    if (nonDivisibles == num - 2) console.log("The number is prime.")

// If the user enters a negative value, this error is displayed.
} else  console.log("Please enter a valid value.")

File: prime-number-checker/prime.py
try:

    # Getting the required values from the user.
    num = int(input("Enter the number you want to check to be a prime number or not: "))
    nonDivisibles = 0

    # Getting hte factors.
    if num > 1:

        for divisor in range(2, num):

            if num % divisor == 0:

                print("The number is composite.")
                break

            else:

                nonDivisibles += 1

        # Output.
        if nonDivisibles == num - 2:

            print("The number is prime.")

    # Making sure the user entered a positive integer.
    else:

        print("Please enter a positive number.")

# If the user enters a string instead of an integer, then this message is printed out instead of the program crashing with an error.        
except ValueError:

    print("Please enter a valid value.")